app/src/main/java/com/nsc/covidscore/MainActivity.java

```diff
@@ -150,7 +204,6 @@ public void locationSelectToRiskFragment() {
 
     public void openNewRiskDetailPageFragment(CovidSnapshot cs, Location selectedLocation) {
         // Used when opening the app with an existing CovidSnapshot
-
         if (cs.hasFieldsSet() && cs.getLocationId() != null) {
             Log.i(TAG, "openNewRiskDetailPageFragment2: ++ Inserting CS" + cs.toString());
             vm.makeApiCalls(selectedLocation);
```

Comment:
Maybe this is where you should add the code to set the selected/highlighted location in the NavBar? I noticed that if I selected a new location manually, the navbar selection wouldn't update.

Reply:
Either this was fixed in the other changes you requested, or I don't comprehend the bug you've described. I'll push commits and follow up with you

Reply:
I think I know what Larissa is talking about.
1. Add three locations
2. Pick a new location
3. See that the highlighted location doesn't match the location on the risk fragment

e.g. in the screenshot below I chose Sussex, Delaware as a new location but Bent, Colorado is the one highlighted when I open the nav
<img width="399" alt="image" src="https://user-images.githubusercontent.com/30481844/99467682-c576e700-28f3-11eb-8c42-c24495c31d53.png">

Reply:
Thanks Dan! I'll fix it now

Reply:
A solution has been pushed

Reply:
I'm still having the same issue

Reply:
Changes by @unicornsleuth have been merged. Should be solved.